crates/swc_ecma_minifier/tests/fixture/issues/6864/output.js

```diff
@@ -0,0 +1,22 @@
+export function removeFromMatrix(matrix, id) {
```

Comment:
[The output of `v1.3.28`](https://play.swc.rs/?version=1.3.28&code=H4sIAAAAAAAAA31STWuDQBC9F%2FofpjcFkabHpAZ6aKGHUAi9iQQbx2RBZ8u6RqXxv3fdNbqatJ7c8X3NW7H%2B5kJCWtJeMk4gMOcnfBM838RSsNq5vwP15PqwhC3GCaeseREibp6np0JB6LBee4bCkiWYkRq48GOmp1gAYbXp9f7TWY0MRgnWH%2Bl78sm3OuASqMy%2FUCjMiBK8mifstSCAnZ8qFccs4oGDJEXjGWUXgvUloI6egrPzGe2zMsFigCaua6M0ch5MOenZaooTKEtBoJWsT615bd1hj876QS0ycerZJntPb22GIvgZ0kEeIQgCWEzYQ926hYrJIy%2FlUETnNbhf9AZK%2BBjZwk9XBew5FbL7a2JGqunXbkHlYwvMqrDjhOGUqdCRB%2FPhIoqiG6UBZgX%2BtWjo%2B75Z0aaOsa4uLpq0o1rRFz4vu7%2BJ0el8Hq%2Bl%2FQVQDhftTQMAAA%3D%3D&config=H4sIAAAAAAAAA22TQZLjIAxF78I6i6ksZpEDzK7PQBEQDj0YuZBIx5XK3UfYMcTJ7OD5S5a%2BxF19k1Wnu5pMJsj1RHNic1MnxfMEZHOYWB0UkyDOBR5yNnkAFgXQ8dfxKF8jIoE6eRMJDmoMKfi55rI4ThmI6lmCygiJqelMzvhDa9qDOiNGMOnjrg3pkBgGyD3UYoxmItBXk1tE%2FZvJgTDtUGFweso4vdDkAgdMkmxjDozTFh00EDJYDlfoEskhkkRSWKtkgQ7OZRiqf6sSriYWwy0X3BYb5I8t7oKBWPuS6A3tKl3R2uRTFbzOwCWnTfONIe18%2BAsgdUZDlMwIPXLhXuawV%2Fr%2FqELyMkSeG5UJ96oSDNKcDsG3DmvVkDl0tzK4YqF2aHvyJ9w1RMGBBu%2FF7RZMP4HtpSeru4i%2BXcU14%2FsM1qve9mlP6wp8wD9SFb%2BO48lHw5d3RvN4xviRYgS%2BoPvAYgTjO8yyvbfpnZbkQGwG9%2FKhUL22NZXhM%2BpYX9vmsyyExOgh4rmtrzzK0aShreWjAnSlgvti3vJaf6tHf53bOtHXU7gk%2BgcCGYLtEQQAAA%3D%3D)